main.js

```diff
@@ -0,0 +1,112 @@
+
+function justDoIt (name){
+    return name + "justDoIt";
+}
+console.log(justDoIt+" "+("Rawan"));
+
+//part1 
+function bigOrSmallString (name){ 
+    if (name.length>10){ 
+        return name + "word is loooooong";
+        console.log("word is loooooong")
+
+    }else{
+        return name = "This word is short";
+        console.log("This word is loooooong!")
+    }
+}
+//part 2 
+function oddOrEvenString (string){
+    if (string % 2==o) {
+    return "This string length is TOTALLY even!";
+    console.log("This string length is TOTALLY even!")
+    }else{
+        "This string's length is odd"
+        console.log(This string's length is odd..")
+}
+
+}
+//part3 
+function medianOfArray (array)
+   var array = [1,2,3,4,5,6,7,8,9,10];
+
+}
+console.log()
+
+
+//part4 //$a=array
+function sumArray (array){
```

Comment:
I don't think you understand how functions work please remind me to explain it further if you want to